prompt/prompt-executor/prompt-executor-clients/prompt-executor-anthropic-client/src/commonMain/kotlin/ai/koog/prompt/executor/clients/anthropic/AnthropicLLMClient.kt

```diff
@@ -44,9 +44,32 @@ import kotlin.uuid.Uuid
 public class AnthropicClientSettings(
     public val modelVersionsMap: Map<LLModel, String> = DEFAULT_ANTHROPIC_MODEL_VERSIONS_MAP,
     public val baseUrl: String = "https://api.anthropic.com",
-    public val apiVersion: String = "2023-06-01",
+    public val apiVersion: String = DEFAULT_ANTHROPIC_API_VERSION,
     public val timeoutConfig: ConnectionTimeoutConfig = ConnectionTimeoutConfig()
-)
+) {
+    /**
+     * Contains default values for the [AnthropicClientSettings] parameters
+     * */
+    public companion object {
+        /**
+         * Default versions map for anthropic versions. See [AnthropicClientSettings.modelVersionsMap] for more information
+         * */
+        public val DEFAULT_ANTHROPIC_MODEL_VERSIONS_MAP: Map<LLModel, String> = mapOf(
```

Comment:
Am I right in understanding that this needs to be made public for koog-ktor?

Reply:
It was made public before I took over, I made some changes that make this unnecessary.